crowdin.yml

```diff
@@ -0,0 +1,3 @@
+files:
+  - source: /templates/locales/strings.yaml
```

Comment:
can we use _/templates/locales/en.yaml_ for the source? so we don't have to maintain 2 files for same purpose?

Reply:
nop, I changed this on purpose.
By the way, we don't need to maintain the file en.yaml which will be maintained from crowdin